src/assets/AssetsDetectionController.ts

```diff
@@ -62,12 +61,12 @@ export class AssetsDetectionController extends BaseController<AssetsDetectionCon
   private async getOwnerCollectibles() {
     const { selectedAddress } = this.config;
     const api = this.getOwnerCollectiblesApi(selectedAddress);
-    const assetsController = this.context.AssetsController as AssetsController;
     let response: Response;
     try {
+      const openSeaApiKey = this.getOpenSeaApiKey();
```

Comment:
This was moved to a getter to make it easier to inject. This _could_ be done in a separate PR :thinking:

Reply:
Nah, I'll leave it. It seems like a bit of a silly change in isolation.